extensions/cargo/PhylumExt.toml

```
@@ -0,0 +1,10 @@
+name = "cargo"
+description = "Cargo package manager hooks"
+entry_point = "main.ts"
+
+[permissions]
+write = ["./", "~/.cargo", "~/.local/share/phylum/extensions/cargo"]
```

This directory isn't needed anymore

```suggestion
write = ["./", "~/.cargo"]
```